webapp/channels/src/components/__snapshots__/searchable_channel_list.test.tsx.snap

```diff
@@ -65,7 +65,7 @@ exports[`components/SearchableChannelList should match init snapshot 1`] = `
   </div>
   <div
     className="more-modal__list"
-    role="application"
+    role="search"
```

Comment:
This fixes the issue in the screenshot.
<img width="537" alt="Screenshot 2023-07-26 at 4 34 28 PM" src="https://github.com/mattermost/mattermost/assets/5334504/649e185c-51f6-4cc2-b677-69bd494a4362">